README.md

```diff
@@ -153,7 +153,8 @@ require'nvim-tree'.setup { -- BEGIN_DEFAULT_OPTS
     },
     icons = {
       webdev_colors = true,
-    },
+      git_placement = "before"
```

Comment:
I see `git_placement = "sign_column"` in the future.

Reply:
:+1: